backend/src/main/java/org/sefglobal/core/controller/MultiverseController.java

```diff
@@ -59,7 +57,18 @@ public University updateUniversity(@PathVariable long id,
         }
     }
 
-    //todo: Implement an endpoint to delete university (change active status)
+    @DeleteMapping("/universities/{id}")
+    public University deleteUniversity(@PathVariable long id) throws ResourceNotFoundException {
+        if (universityRepository.existsById(id)) {
+            University university = universityRepository.getOne(id);
+            university.setStatus("REMOVED");
```

Comment:
Add status values to a constants class and refer from there instead of hardcoding